tensorbay/client/commit_status.py

```diff
@@ -73,21 +75,23 @@ def check_authority_for_commit(self) -> None:
         """Check whether the status is a legal commit.
 
         Raises:
-            TypeError: When the status is not a legal commit.
+            CommitStatusError: When the status is not a legal commit.
 
         """
+        required_status = "commit"
```

Comment:
Use `str` as status is not stable, is that possible to use `bool` or `Enum` even use `CommitStatus` class as the status?